hw/ip/clkmgr/data/clkmgr.sv.tpl

```diff
@@ -408,15 +434,27 @@
     .lc_en_o(${clk}_scanmode)
   );
 
+  logic ${clk}_combined_en;
```

Comment:
i can't completely remember this now..but given this and the above are both `combined_en`, could we potentially have name conflicts?

Reply:
o one thing i forgot, should we buffer the enable signals to ensure the one sent outside and to the CG are always the same?

Reply:
let me double check that. good point RE buffering, though! let me add that in.

Reply:
RE name conflicts, I used this label since it is already used in the code to uniquify signals (see for instance on line 400 above (`lc_tx_t ${clk}_scanmode;`). so I think this should be fine.

I also added a buffer instance here. 
PTAL